xcode/Subconscious/SubconsciousTests/Tests_Slashlink.swift

```diff
@@ -233,7 +233,26 @@ final class Tests_Slashlink: XCTestCase {
             "Does not relativize when did is other sphere"
         )
     }
-
+    
+    func testRelativizeIfNeededNilDid() throws {
+        let did = try Did("did:key:z6MkmCJAZansQ3p1Qwx6wrF4c64yt2rcM8wMrH5Rh7DGb2K7")
```

Comment:
Is this first `did` variable used in the test?

Reply:
Fixed